lib/input.rb

```diff
@@ -0,0 +1,9 @@
+class Input
+    def prompt_input
+        "Please make a selection"
+    end
```

Comment:
This function can/should probably just be a constant. Constants are good to use when you have some data that won't change, since they can't be mutated.

Reply:
Thanks, I'm doing some syntax research.